### PR TITLE
Dataset split

### PR DESCRIPTION
**Pull request by majsylw**
add multilabel stratified split

### CODE REVIEW

**Comment by majsylw** [comment]
Using stratify split improve loss at the begining of training (red and blue lines), but then the losses reach the same level as for random splitting (grin and violet lines).
image

File: datasets/dataset_creator.py [hunk 1]
```diff
@@ -1,19 +1,49 @@
...
+from torch.utils.data import DataLoader
+
+from datasets.dataset import PadCollate, VideoFrameDataset
 from datasets.transforms_creator import TransformsCreator
 
 
+def iterative_train_test_split(X, y, train_size):
+    """Custom iterative train test split which
+    'maintains balanced representation with respect
+    to order-th label combinations.'
```
Comment (wiktorfilipiuk) [inline datasets/dataset_creator.py:21]: Do we need apostrophes in the docstring?

**Review by wiktorfilipiuk — COMMENTED** [review]
Rebase with current version necessary :)

**Comment by majsylw** [comment]
Discarded due end of the project
